throw(new Error("这是我自己抛出的错误"))